src/TheSpace/TheSpace.sol

```diff
@@ -52,26 +64,16 @@ contract TheSpace is HarbergerMarket {
     /**
      * @notice Get pixel info.
      */
-    function getPixel(uint256 tokenId_)
-        external
-        view
-        returns (
-            uint256 tokenId,
-            uint256 price,
-            uint256 lastTaxCollection,
-            uint256 ubi,
-            address owner,
-            uint256 color
-        )
-    {
-        return (
-            tokenId_,
-            tokenRecord[tokenId_].price,
-            tokenRecord[tokenId_].lastTaxCollection,
-            ubiAvailable(tokenId_),
-            getOwner(tokenId_),
-            pixelColor[tokenId_]
-        );
+    function getPixel(uint256 tokenId_) external view returns (Pixel memory) {
```

Comment:
can add an internal `_getPixel` that is reused by `getPixel` and `getPixelsPageByOwner`